### DISCUSSION TITLE
How to preserve #if DEBUG #endif from original template?

### DISCUSSION THREAD

**Discussion by geocine** [dotnet/templating#6478] [Template authoring] [answered]
I created a template with `#if DEBUG` then I created a new project out of that template. I noticed on the new project, I no longer see the `#if DEBUG` . Is it possible to preserve that?

**Comment by GangWang01** [comment, marked as answer]
[Ignore conditions expressions in language files](https://github.com/dotnet/templating/wiki/Conditional-processing-and-comment-syntax#ignore-conditions-expressions-in-language-files) could help on your need.
